ruby3.2-ffi.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: ruby3.2-ffi
+  version: 1.17.0
+  epoch: 0
+  description: Ruby FFI
+  copyright:
+    - license: BSD-3-Clause And MIT
+  dependencies:
+    runtime:
+      - autoconf
+      - automake
+      - libffi-dev
+      - libtool
+      - ruby-3.2
+      - ruby3.2-bundler
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
```

Comment:
`automake` and `autoconf` are only needed if used in the build process. We should be able to get rid of these